tasks/build-secrets

```diff
@@ -16,8 +16,10 @@ metadata:
 data:
 EOF
 cd "$BASE/tasks"
-for f in $(find -type f -o -type l); do
-    printf '  %s: %s\n' "$(echo $f | sed 's!^./!!; s!/!--!g')" "$(base64 --wrap=0 $f)"
+# This intentionally does not capture subdirs like tasks/s3-keys/. As OpenShift secret volumes don't have subdirectories,
+# these need to be created as a separate volume if and when we ever need that.
+for f in $(find -maxdepth 1 -type f -o -type l); do
+    printf '  %s: %s\n' "${f#./}" "$(base64 --wrap=0 $f)"
```

Comment:
I was going to suggest "that thing you can do with shell parameter expansion that drops prefixes" as another option, but I was too lazy to look it up ;)